### ISSUE TITLE
Image package

### ISSUE THREAD

**Issue by WanWizard** [fuelphp/fuelphp#10]
I'm not convinced the 1.x Image class should stay. It's part voodoo, it's part a coding mess, and nobody knows exactly how it works, which makes maintaining difficult.

Perhaps we should drop it, and suggest http://imagine.readthedocs.org/en/latest/ instead. There's a composer package for it. Not sure it's possible to create a 1.x compatibility class for it, and not sure either it covers all functionality.

I'm open to ideas!

**Comment by sagikazarmark** [comment]
http://image.intervention.io/ might worth a look as well. Seems to be easier to use. However I am not even sure we should provide one. The old API is too much complex, both of the mentioned libraries are simple and heavily object oriented.

**Comment by stevewest** [comment]
@sagikazarmark makes a good point, it might not be worth us implementing an image library given that there are quite a few now out there. What might be a good idea is looking at some libraries that we like the look of and submit PRs with fuel service providers for "integration" into the framework.

**Comment by sagikazarmark** [comment]
Both of the mentioned ones have laravel providers, so I think submitted PRs might be accepted. If not, we still can create integration packages.

**Comment by stevewest** [comment]
Works for me then. I don't really fancy the idea of creating yet another image package.

**Comment by sagikazarmark** [comment]
Yeah. Not to mention, that in the age of packagist, it is super easy to install one or another based on your needs. So besides core "MVC" (application and some extra like orm, auth) features we don't really have to provide anything else (which has been specifically solved already).

**Comment by WanWizard** [comment]
I don't think it was ever the intention to create one. Possibly (as part of the legacy package) porting the current v1 class for BC reasons...
